src/components/LicensePicker/utils.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
```

Comment:
This file is really a defining an spdx expression class. In other places we have a mess of utilities etc for dealing with expressions. it would be useful to gather all of these together into one class and put that out as an npm.

Don't need to do anything for this PR but it would be an interesting thing to do. @dabutvin, what do you think?

Reply:
@jeffmcaffer i agree, specially if some of these things are shared with the service too (like the parse method)